docs/book/src/developer/providers/v1.1-to-v1.2.md

```diff
@@ -31,4 +31,9 @@ in ClusterAPI are kept in sync with the versions used by `sigs.k8s.io/controller
 
 ### Other
 
--
+- Logging:
+    - To align with the upstream Kubernetes community CAPI now configures logging via `component-base/logs`. 
+      This provides advantages like support for the JSON logging format (via `--logging-format=json`) and automatic
+      deprecation of klog flags aligned to the upstream Kubernetes deprecation period. For more details please see:
+      [#6072](https://github.com/kubernetes-sigs/cluster-api/pull/6072) and [#6190](https://github.com/kubernetes-sigs/cluster-api/pull/6190).
```

Comment:
Instead of linking to PRs one built on top of the other can we add a diff snippet with all the changes to e.g. main.go?

Reply:
Updated. I added a diff with the most important changes. I still kept links to the PRs around for changes like go.mod, imports, ... . So folks can look those details up if needed.